src/clr-angular/image/_icons.clarity.scss

```diff
@@ -21,6 +21,14 @@ $svg_data: "data:image/svg+xml;charset=utf8";
     @return url("#{$svg_data},#{$warning-icon}");
 }
 
+@function generateCheckIcon($fillColor:$gray-dark) {
```

Comment:
Wait, why are we not using a `<clr-icon>` here? We're trying _not_ to have any encoded SVGs in the CSS anymore.

Reply:
I'm making use of existing `spinner`, `spinner-inline` classes and it makes it a breaking change to get rid of using SVG completely in spinner (or alternatively, gets pretty messy trying to make it an exception for the checkbox case here only). Would it be ok to create a separate issue for getting rid of encoded SVGs in spinner-related components all together in the future?

Reply:
If there is a deprecation we want to make, let's get a ticket in to have that done for 0.12.

Reply:
issue created, though not sure this is amust for 0.12: https://github.com/vmware/clarity/issues/2241